Common Tools/TapticSupport.swift

```diff
@@ -0,0 +1,126 @@
+//
+//  TapticSupport.swift
+//  HSRPizzaHelper
+//
+//  Created by Bill Haku on 2023/5/8.
+//  Taptic Engine支持
+
+import Foundation
+import UIKit
+#if !os(watchOS)
+import AudioToolbox
+import CoreHaptics
+
+enum SimpleTapticType {
+    case success
+    case warning
+    case error
+    case light
+    case medium
+    case heavy
+    case rigid
+    case soft
+    case selection
+}
+
+func simpleTaptic(type: SimpleTapticType) {
```

Comment:
SwiftLint: Cyclomatic Complexity Violation: Function should have complexity 10 or less; currently complexity is 12 (cyclomatic_complexity)